### ISSUE TITLE
"unlimited.cpp:56:36: error: init_priority attribute requires integer constant between 101 and 65535 inclusive" on FreeBSD

### ISSUE THREAD

**Issue by BlinkyStitt** [gandrewstone/bitcoin#39]
After doing the fixes in https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues/38, I ran into another error.

```
unlimited.cpp:56:36: error: init_priority attribute requires integer constant between 101 and 65535 inclusive
CStatMap statistics __attribute__((init_priority(2)));
                                   ^             ~
1 error generated.
Makefile:4150: recipe for target 'libbitcoin_server_a-unlimited.o' failed
gmake[2]: *** [libbitcoin_server_a-unlimited.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[2]: Leaving directory '/usr/home/bitcoin/src/BitcoinUnlimited/src'
Makefile:7073: recipe for target 'all-recursive' failed
gmake[1]: *** [all-recursive] Error 1
gmake[1]: Leaving directory '/usr/home/bitcoin/src/BitcoinUnlimited/src'
Makefile:653: recipe for target 'all-recursive' failed
gmake: *** [all-recursive] Error 1
```

So I changed that 2 to a 101 and am trying again. That line is the only place that init_priority is set so I think that will work fine

**Comment by BlinkyStitt** [comment]
Either that change is wrong or I'm missing something else because gmake succeeded, but gmake check failed:

```
==================================================
   Bitcoin Unlimited 0.12.0: src/test-suite.log
==================================================

# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: test/test_bitcoin
=======================

Running 183 test cases...
test/transaction_tests.cpp(339): error in "test_IsStandard": check nDustThreshold == 546 failed [0 != 546]
unknown location(0): fatal error in "DoS_banning": memory access violation at address: 0x00000008: no mapping at fault address
test/getarg_tests.cpp(159): last checkpoint

*** 2 failures detected in test suite "Bitcoin Test Suite"
Database handles still open at environment close
Open database handle: unnamed/wallet.dat
FAIL test/test_bitcoin (exit status: 201)
```

Think that is related to this change or my change in #38, or should I open a separate issue for this one?

**Comment by BlinkyStitt** [comment]
Latest master seems fixed:

```
============================================================================
# TOTAL: 1
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

**Comment by BlinkyStitt** [comment]
Silly me. "Latest master" is not BU. This was still an issue, but it got fixed in deb75a19
